### PR TITLE
Bug 1220989 - migrate communications app to notificationHelper

### PR DESCRIPTION
**Pull request by hkasemir**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cccd4bdfc2a53a23e0d66919f4de947032f29422

**Comment by zbraniecki** [comment]
You should also look into `./apps/communications//facebook/js/fb_sync_init.js`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=312e76fff903a065c5551b8766d6629f64b36b56

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=017f7938bc8a123d3aaf47e4a39a91be535e0dc2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=59ea7d41830f60a0839d68fb79a1fea758ae4ae4

File: apps/communications/dialer/js/mmi.js [hunk 1]
```diff
@@ -356,19 +356,20 @@ var MmiManager = {
             };
             var conn = navigator.mozMobileConnections[cardIndex || 0];
             var operator = MobileOperator.userFacingInfo(conn).operator;
-            var title = self.prependSimNumber(operator ? operator : '',
+            var titleL10n = self.prependSimNumber(operator ? operator : '',
```
Comment (gabrielesvelto) [inline apps/communications/dialer/js/mmi.js]: nit: Please move the arguments to the line below, that'll make the formatting nicer, e.g.

```javascript
var titleL10n = self.prependSimNumber(
  operator ? operator : '', cardIndex
);
```

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5cbe8d214d453e99900553336bda4f4576c1b313

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d7173f7d6aa9242a8d80724c79a579201ceed6a9

**Pull request merged**